cypress/integration/racingcar.spec.js

```diff
@@ -0,0 +1,145 @@
+describe('자동차 경주 게임', () => {
+  beforeEach(() => {
+    cy.visit('../index.html');
+  });
+
+  it('앱을 처음 접속하면 자동차 이름을 입력하는 화면만 보여야 한다.', () => {
+    // given
+    // when
+    // then
+    cy.get('#racing-try-count').should('not.be.visible');
+    cy.get('#racing-track').should('not.be.visible');
+    cy.get('#racing-result').should('not.be.visible');
+  });
+
+  it('자동차 이름을 입력하기 위한 input 과 button이 렌더링 되었는지 확인한다.', () => {
+    cy.get('#car-names-input').should('be.visible');
+    cy.get('#car-names-submit').should('be.visible');
+  });
+
+  it('자동차 이름을 입력하지 않으면 경고창 메시지를 보여준다.', () => {
+    const alertStub = cy.stub();
+    cy.on('window:alert', alertStub);
+
+    cy.get('#car-names-input').clear();
+    cy.get('#car-names-submit')
+      .click()
+      .then(() => {
+        expect(alertStub).to.be.called;
+      });
+  });
+
+  it('자동차 이름이 6글자 이상이면 경고창 메시지를 보여준다.', () => {
+    const alertStub = cy.stub();
+    cy.on('window:alert', alertStub);
+
+    cy.get('#car-names-input').type('EVERYDAY');
+    cy.get('#car-names-submit')
+      .click()
+      .then(() => {
+        expect(alertStub).to.be.called;
+      });
+  });
+
+  it('시도 횟수 입력창은 자동차 이름을 제출한 후에 표시된다.', () => {
+    // given
+    cy.get('#racing-try-count').should('not.be.visible');
+
+    // when
+    cy.get('#car-names-input').type('CHILL,HIP');
+    cy.get('#car-names-submit').click();
+
+    // then
+    cy.get('#racing-try-count').should('be.visible');
+  });
+
+  it('시도 횟수가 0 이하일 경우 경고창을 호출한다.', () => {
+    // given
+    cy.get('#racing-try-count').should('not.be.visible');
+    cy.get('#car-names-input').type('CHILL,HIP');
+    cy.get('#car-names-submit').click();
+    cy.get('#racing-try-count').should('be.visible');
+
+    // when
+    cy.get('#try-count-input').type('-1');
```

Comment:
```suggestion
    const WRONG_INPUT = -1;
    
    // given
    cy.get('#racing-try-count').should('not.be.visible');
    cy.get('#car-names-input').type('CHILL,HIP');
    cy.get('#car-names-submit').click();
    cy.get('#racing-try-count').should('be.visible');

    // when
    cy.get('#try-count-input').type(WRONG_INPUT);
```
-1 그리고 다른테스트에서 쓰인 시도횟수도 변수에 담아서 사용하면 더 가독성 좋은 테스트가 될거같아요.

Reply:
이 내용은 step2 에서 반영하겠습니다!